### ISSUE TITLE
Retrying setup: '>' not supported between instances of 'NoneType' and 'float'

### ISSUE THREAD

**Issue by dhover** [briis/hass-weatherflow#12]
I get this error directly after adding the integration version 0.1.11

![error](https://user-images.githubusercontent.com/34840601/144887367-42f24c6c-5dfe-4719-9895-47a76db69d1c.jpg)

In the logfile:

```
2021-12-06 17:47:36 ERROR (MainThread) [custom_components.weatherflow] Unexpected error fetching weatherflow data: '>' not supported between instances of 'NoneType' and 'float'
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
self.data = await self._async_update_data()
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 147, in _async_update_data
return await self.update_method()
File "/config/custom_components/weatherflow/__init__.py", line 105, in async_update_data
data: ObservationDescription = await weatherflowapi.update_observations()
File "/usr/local/lib/python3.9/site-packages/pyweatherflowrest/api.py", line 191, in update_observations
beaufort_data: BeaufortDescription = self.calc.beaufort_value(obervations.get("wind_avg"))
File "/usr/local/lib/python3.9/site-packages/pyweatherflowrest/helpers.py", line 266, in beaufort_value
if wind_speed > float(k):
TypeError: '>' not supported between instances of 'NoneType' and 'float'
2021-12-06 17:47:36 DEBUG (MainThread) [custom_components.weatherflow] Finished fetching weatherflow data in 1.144 seconds (success: False)
2021-12-06 17:47:36 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Weerstation Markelo' for weatherflow integration not ready yet: '>' not supported between instances of 'NoneType' and 'float'; Retrying in background
```

**Comment by dhover** [comment]
Maybe I know what the problem is.
I see my station is in power saving mode.
Battery voltage is 2.35 V.
In  mode 3 wind sampling is increased to 5 minutes.

```
Power Save Modes
Here are details on how the Tempest may conserve power in order to operate for as long as possible. Most Tempests never go below Mode 0 and more than 90% will never go below Mode 1.

Mode 0: Voltage ≥ 2.455

All sensors enabled and operating at full performance
Wind sampling interval every 3 seconds
Mode 1: Voltage ≤ 2.415 from Mode 0 or ≥  2.41 from Mode 2

Wind sampling interval set to 6 seconds
Mode 2: Voltage ≤ 2.39 from Mode 1 or ≥  2.375 from Mode 3

Wind sampling interval set to one minute
Mode 3: Voltage ≤ 2.355

Wind sampling interval set to 5 minutes
All other sensors' sampling interval set to 5 minutes
Haptic Rain sensor disabled from active listening
```
Could that be the problem?
The old integration is working fine but all wind measurements are zero.

![error](https://user-images.githubusercontent.com/34840601/144893189-856e4730-4164-48c1-9d0d-fce73a8782c7.jpg)

**Comment by briis** [comment]
I can see you are from Holland, and I have the same issue here in DK, when we don't get enough sunlight to charge the battery. I still thought I had fixed this issue in .11 but I will give it another go, so that the setup does not fail.

**Comment by dhover** [comment]
Can I use both integrations at the same time?

**Comment by briis** [comment]
If you mean SmartWeather and this one, the answer is yes.

Btw: Found why it crashed, and I am now just returning empty values, if wind_avg is not present, so it will startup nicely and just supply values when/if they are present. Will be included in next release.

**Comment by dhover** [comment]
Good news. I'll test it when the release is available.

**Comment by briis** [comment]
Will be sometime tomorrow.

**Comment by briis** [comment]
Released V0.1.12 which fixes this issue. Re-Open if it does not work.
